CONTRIBUTING.md

```diff
@@ -153,66 +180,127 @@ Sometimes, it might not make sense for ECS to standardize a field, value, or cat
+rule_id [86af1430-c0cd-428c-96e3-964268645120] ("n/a" to leave blank)  (required):
 actions (multi, comma separated):
-description (required): Look for child processes of MsBuild
+description (required):
 enabled [false] ("n/a" to leave blank):
 from [now-6m] ("n/a" to leave blank):
 false_positives (multi, comma separated):
 filters (multi, comma separated):
 interval [5m] ("n/a" to leave blank):
-exceptions_list (multi, comma separated):
 max_signals [100] ("n/a" to leave blank):
 meta:
-name (required): Suspicious Child of MsBuild
+name (required):
 note:
 references (multi, comma separated):
 risk_score [21] ("n/a" to leave blank)  (required):
-rule_id [90d0c543-e197-46d8-934d-0320b2c83486] ("n/a" to leave blank)  (required):
-severity [low] ("n/a" to leave blank)  (required): medium
-tags (multi, comma separated): Windows  
+severity [low] ("n/a" to leave blank)  (required):
+tags (multi, comma separated):
 throttle:
 timeline_id:
 timeline_title:
 to [now] ("n/a" to leave blank):
-threat (multi, comma separated):
-index (multi, comma separated): winlogbeat-*
-language [kuery] ("n/a" to leave blank)  (required): kuery
-query (required): event.category:process and process.parent.name:msbuild.exe
-ecs_version (multi, comma separated): 1.4.0
+add mitre tactic? [y/N]:
+index (multi, comma separated):
+language [kuery] ("n/a" to leave blank)  (required):
+query (required):
+author (required)  (multi, comma separated):
+building_block_type:
+exceptions_list (multi, comma separated):
+license [Elastic License] ("n/a" to leave blank)  (required):
+risk_score_mapping (multi, comma separated):
+rule_name_override:
+severity_mapping (multi, comma separated):
+timestamp_override: N
+mitre tactic name (required):
+technique IDs for Command and Control (required)  (multi, comma separated):
+add mitre tactic? [y/N]:
 ```
 
-Pending no errors, you'll see this output upon success
+1. You will receive this output when successful:
+
 ```
-Rule Suspicious Child of MsBuild saved to rules/windows/defense_evasion_msbuild_child.toml
+Rule saved to /PATH/TO/detection-rules/RULE.toml
 Did not set the following values because they are un-required when set to the default value
  - from
  - interval
  - max_signals
  - to
+Rule will validate against the latest ECS schema available (and beats if necessary)
+    - to have a rule validate against specific ECS schemas, add them to metadata->ecs_versions
+    - to have a rule validate against a specific beats schema, add it to metadata->beats_version
 ```
 
 
-### Testing a rule with the CLI
+### Importing rules
+
+1. As shown above, rules can be created in an appropriate directory with `rules/`. ***If*** rules are created outside of this structure, you can import them into the project repo with the `import-rules` sub-command.
+
+1. For example, this command will copy any newly-developed rule files into `rules/` directory:
+
+```shell
+    python -m detection_rules import-rules -d ~/dev-rules/
+```
```

Comment:
This section might be redundant to the `import-rules` section of the CLI.md. If having it here is useful, maybe just a reference to that section would suffice